doc/src/modules/combinatorics/fp_groups.rst

```diff
@@ -0,0 +1,154 @@
+Introduction
+============
+
+This module presents the functionality designed for computing with finitely-
+presented groups (fp-groups for short). The name of the corresponding SymPy
+object is ``FpGroup``. The functions or classes described here are studied
+under **computational group theory**.
+
+Overview of Facilities
+----------------------
+
+The facilities provided for fp-groups fall into a number of natural groupings
+
+* The construction of fp-groups using a free group and a list of words in
+  generators of that free group.
+
+* Index determination using the famous Todd-Coxeter procedure.
+
+* The construction of all subgroups having index less than some (small)
+  specified positive integer, using the *Low-Index Subgroups* algorithm.
+
+For a description of fundamental algorithms of finitely presented groups
+we often make use of *Handbook of Computational Group Theory*.
+
+The Construction of Finitely Presented Groups
+---------------------------------------------
+
+Finitely presented groups are construced by factoring a free group by a
+set of relators. The set of relators is taken in as a list of words in
+generators of free group in SymPy, using a list provides ordering to the
+relators. If the list of relators is empty, the associated free group is
+returned.
+
+Example of construction of a finitely-presented group.
+The symmetric group of degree 4 may be represented as a two generator group
+with presentation `< a, b | a^2, b^3, (a*b)^4 >`. Giving the relations as a
```

Comment:
It seems that `:math:` is not needed, single backquotes alone suffice. The angle brackets could probably be `\langle` and `\rangle`. The `*` in the product `ab` is not necessary here in the documentation since it is not intended to be machine readable.